### PR TITLE
chore: add integration tests for user language settings

### PR DESCRIPTION
**Pull request by kurokochin**
### Feature Preview

This PR add integration tests for user language settings.
Solve #3416 

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/integration_test/settings/user_language_test.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      await tester.openSettingsPage(SettingsPage.language);
+
+      final userLanguageFinder = find.descendant(
+        of: find.byType(SettingsLanguageView),
+        matching: find.byType(LanguageSelector),
+      );
+
+      // Grab current locale
+      LanguageSelector userLanguage =
+          tester.widget(userLanguageFinder) as LanguageSelector;
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/integration_test/settings/user_language_test.dart:29]:
```suggestion
      LanguageSelector userLanguage =
          tester.widget<LanguageSelector>(userLanguageFinder);
```

**Comment by LucasXu0** [comment]
Hi, @kurokochin. Please add your test to `runner.dart`, otherwise the CI won't execute it.

**Comment by kurokochin** [comment]
> Hi, @kurokochin. Please add your test to `runner.dart`, otherwise the CI won't execute it.

Sure thing! I noticed that `user_icon_test.dart` also not there, should I add it to `runner.dart` too?

**Comment by LucasXu0** [comment]
> user_icon_test

Yes. Thanks.

**Review by richardshiue — APPROVED** [review]
LGTM :shipit:

**Pull request merged**